paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/playground/model/PlaygroundCheckoutModel.kt

```diff
@@ -107,20 +116,27 @@ data class CheckoutResponse(
     val customerId: String? = null,
     @SerialName("customerEphemeralKeySecret")
     val customerEphemeralKeySecret: String? = null,
+    @SerialName("customerSessionClientSecret")
+    val customerSessionClientSecret: String? = null,
     @SerialName("amount")
     val amount: Long,
     @SerialName("paymentMethodTypes")
     val paymentMethodTypes: String? = null,
 ) {
-    fun makeCustomerConfig() =
-        if (customerId != null && customerEphemeralKeySecret != null) {
+    @OptIn(ExperimentalCustomerSessionApi::class)
+    fun makeCustomerConfig() = customerId?.let { id ->
```

Comment:
You should be able to pass the type here, and choose the right one based on that.